glyphs-reader/src/font.rs

```diff
@@ -982,6 +996,17 @@ impl RawLayer {
         self.associated_master_id.is_some() && self.attributes == Default::default()
     }
 
+    fn is_bracket_layer(&self, format_version: FormatVersion) -> bool {
```

Comment:
a comment explaining what a bracket layer *is* would be nice, bit of an obtuse concept